docs/example-apps/rails.md

```diff
@@ -1,45 +1,30 @@
 # Rails
 
-Our example Rails app can be found [here](https://github.com/convox-examples/rails).  You can clone this locally to run and experiment with.
+You can find an example Rails app [here](https://github.com/convox-examples/rails).  You can clone this repository locally to run and experiment with it.
 
-### Running Locally
+### Preparing your environment
 
 Before we begin you will need to install the `convox` CLI and a development Rack:
 
 * [Command Line Interface](../installation/cli.md)
 * [Development Rack](../installation/development-rack)
 
-Once you are all setup you can switch to your local rack with ```convox switch [rack name]``` and start your local application with ```convox start``` (make sure you are in the root directory).
-
-You should now be able to access your application by going to [https://web.rails.convox](https://web.rails.convox). If you renamed anything you may need to modify your local URL. The format is https://[service name].[app name].convox
-
-### Custom Application Components
-
-#### [Dockerfile](https://github.com/convox-examples/rails/blob/master/Dockerfile)
-
-Starting from the [ruby:2.6.4](https://hub.docker.com/_/ruby/) image, the `Dockerfile` defines the steps necessary to turn the application code into an image that is ready to run. 
-
-This `Dockerfile` has 3 steps and they are executed in a particular order to take advantage of Docker's build caching behavior.
-
-1. `bundle install` and `yarn install` are run to install dependencies after copying just the files needed to run these commands. This will ensure that the output these commands are cached unless one of these files changes.
-
-2. The application source is copied over. These files will change frequently so this step of the build will very rarely be cached.
-
-3. Finally, after setting the appropriate environment variables the assets are precompiled.
-
-#### [convox.yml](https://github.com/convox-examples/rails/blob/master/convox.yml)
+### Preparing your application
 
-The `convox.yml` manifest explains how to run the application. The manifest for this application has two sections:
-
-1. Resources: These are network-attached dependencies of the application. In this application we have a single resource, a `postgres` database.
+To deploy your rails application in your local rack your need to make a few changes to it. 
 
-2. Services: These are the web-facing services of the application. This application has a single service named `web` which is built from the local directory.
+1. To run your application you will need a [Dockerfile]. Create a file in the root of your project with the name `Dockerfile` and the content of our [example](https://github.com/convox-examples/rails). Notice that ours is using ruby-3.0, if you need a different version feel free to change the version on the first line of the file.
```

Comment:
Image references should be valid references from Docker Hub if they change the version.
Maybe if you're going to remove the section describing what the Dockerfile is doing, then link to Dockerfile docs for people to understand that themselves?

Reply:
@beastawakens my goal here is to provide a docker image that they can use out of the box. We are using an official ruby imager from Dockerhub. I'll add the reference for our Dockerfile docs

Reply:
I guess my point is that if we're assuming the user doesn't know what a Dockerfile is, or how to Dockerise their application, we should be clear with instructions like changing base image version.  Otherwise they may change the version number, and have everything fail straight away because that image doesn't exist in Docker Hub for whatever reason.